test/per_file_coverage.sh

```diff
@@ -57,6 +57,7 @@ declare -a KNOWN_LOW_COVERAGE=(
 "source/extensions/transport_sockets/tls/private_key:76.9"
 "source/extensions/transport_sockets/tls:95.0"
 "source/extensions/wasm_runtime:50.0"
```

Comment:
You need to lower this to `40.0` to pass coverage:
`(NullVM + V8) / (NullVM + V8 + WAMR + Wasmtime + WAVM)` ~= `2 / 5` ~= `40%`

Reply:
Got it. So it seems the coverage-check only check nullvm + v8 under wasm_runtime?

Reply:
Yes, coverage is tested only with default build options (V8 and NullVM).